src/rapala.py

```diff
@@ -1,20 +1,22 @@
 from bs4 import BeautifulSoup as bs
 from selenium import webdriver
-from selenium.webdriver.chrome.options import Options
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.chrome.service import Service
 from selenium.webdriver.remote.webelement import WebElement
 from selenium.webdriver.common.by import By
 import time
+import yaml
 from typing import Any, Dict, List
 
+CONFIG = yaml.load(open("config.yml"), Loader=yaml.FullLoader)
 
 class Rapala:
     def __init__(
         self,
+        firefox_path: str = None,
         chrome_path: str = None,
```

Comment:
Can we just do one variable `browser` and then map the path to the browser type?

Reply:
Yes, It's done but with the variable name _driver_type_, please review it.